src/csharp/Grpc.Examples/MathServiceImpl.cs

```diff
@@ -1,6 +1,6 @@
 #region Copyright notice and license
 
-// Copyright 2015, Google Inc.
+// Copyright 2015-2016, Google Inc.
```

Comment:
I don't *think* the range is required here - I believe you can just leave it as 2015.